GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/RO_BDB_ApolloRevamp.cfg

```diff
@@ -0,0 +1,1358 @@
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.553
+	
+	
+	@title = Lunar Module Ascent Stage
+	@description = Ascent Stage for the Apollo Lunar Module
+	%manufacturer = Grumman
+	@mass = 1.984 //1.786
+	%maxTemp = 673.15
+	%skinMaxTemp = 773.15
+	!RESOURCE,* {}
+	!MODULE[ModuleB9PartSwitch]:HAS[#moduleID[configSwitch]] {}
+	
+	
+	MODULE	//	Quantities copied from the ROCapsules Config by Luci on 1/3/21
+	{
+		name = ModuleFuelTanks
+		type = ServiceModule
+		volume = 2463.7
+		basemass = -1
+
+		TANK
+		{
+			name = ElectricCharge
+			amount = 59675
+			maxAmount = 59675
+		}
+		
+		TANK
+		{
+			name = Aerozine50
+			amount = 1124.6
+			maxAmount = 1124.6
+		}
+		
+		TANK
+		{
+			name = NTO
+			amount = 1139.9
+			maxAmount = 1139.9
+		}
+		
+		TANK
+		{
+			name = Helium
+			amount = 27828.3
+			maxAmount = 27828.3
+		}
+		
+		// Life Support Tanks
+		
+		TANK
+		{
+			name = Food
+			amount = 23.4
+			maxAmount = 35.1
+		}
+
+		TANK
+		{
+			name = Water
+			amount = 40
+			maxAmount = 40
+		}
+
+		TANK
+		{
+			name = Oxygen
+			amount = 1563.4
+			maxAmount = 1563.4
+		}
+
+		TANK
+		{
+			name = LithiumHydroxide
+			amount = 4.1
+			maxAmount = 6.2
+		}
+
+		TANK
+		{
+			name = Waste
+			amount = 0
+			maxAmount = 3.2
+		}
+
+		TANK
+		{
+			name = WasteWater
+			amount = 0
+			maxAmount = 29.3
+		}
+
+		TANK
+		{
+			name = CarbonDioxide
+			amount = 0
+			maxAmount = 1023
+		}
+	}
+}
+
+@PART[bluedog_LM_Descent_Tanks]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.553
+	
+	
+	@title = Lunar Module Descent Stage
+	@description = Descent Stage for the Apollo Lunar Module
+	%manufacturer = Grumman
+	@mass = 1.5
+	%maxTemp = 673.15
+	%skinMaxTemp = 773.15
+	!RESOURCE,* {}
+	
+	
+	MODULE	//	Quantities copied from the ROCapsules Config by Luci on 1/3/21
+	{
+		name = ModuleFuelTanks
+		type = ServiceModule
+		volume = 8163
+		basemass = -1
+
+		TANK
+		{
+			name = ElectricCharge
+			amount = 161280
+			maxAmount = 201600
+		}
+
+		TANK
+		{
+			name = Aerozine50
+			amount = 3544
+			maxAmount = 3791
+		}
+
+		TANK
+		{
+			name = NTO
+			amount = 3520
+			maxAmount = 3765
+		}
+		
+		TANK
+		{
+			name = Helium
+			amount = 75231.6
+			maxAmount = 80471.4
+		}
+	}
+}
+
+@PART[bluedog_LM_Descent_Leg]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.553
+	
+	
+	@title = Lunar Module Leg
+	@description = Landing Legs for the Apollo Lunar Module
+	%manufacturer = Grumman
+	@mass = 0.0815
+	%maxTemp = 673.15
+	%skinMaxTemp = 773.15
+}
+
+@PART[bluedog_LM_Ascent_Engine]:AFTER[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.553
+	
+	!RESOURCE,* {}
+
+	@title = Ascent Propulsion System (LMAE)
+	%maxTemp = 673.15
+	%skinMaxTemp = 773.15
+	%engineType = LMAE
+}
+
+@PART[bluedog_LM_Descent_Engine]:AFTER[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.553
+	
+	!RESOURCE,* {}
+
+	@title = Descent Propulsion System (LMDE)
+	%maxTemp = 1973.15
+	%skinMaxTemp = 3600
```

Comment:
Doesn't seem appropriate for an engine.  Should be in line with all the other engines, which is usually 673/773.